src/Model/EventCommentMapper.php

```diff
@@ -316,11 +317,22 @@ public function getReportedCommentsByEventId($event_id, $moderated = false)
                         . " and ec.ID = :comment_id";
         $comment_stmt = $this->_db->prepare($comment_sql);
 
-        while ($row = $stmt->fetch(PDO::FETCH_ASSOC)) {
+        /**
+         * @var array{
+         *     reporting_user_id: int,
+         *     deciding_user_id: int,
+         *     decision: string,
+         *     event_comment_id: int,
+         *     event_id: int
+         * }|false $row
+         */
+        $row = $stmt->fetch(PDO::FETCH_ASSOC);
+
+        while ($row) {
```

Comment:
hey, since I messed this up, feel free to ping me :) I'll fix it. this happens if you only look blindly at phpstan errors and mix some "tiny" refactoring in...

Reply:
@herndlm PR in the fix and I'll happily take it :). I would've either fixed or reached out but, well, I'm still awake in UTC-5 for work reasons 🙃